src/pages/search.js

```diff
@@ -35,6 +36,14 @@ export default function SearchPage({ location: { pathname, search, hash } }) {
     const [ searchQuery, setSearchQuery ] = useState(urlParams.get("q") || "");
     const [ debouncedSearchQuery ] = useDebounce(searchQuery, 500);
 
+    const pageTitle = (() => {
+        // Generates page title based on search query
+        if (searchQuery && searchQuery.trim()) {
+            return `${searchQuery} - Search`;
+        }
+        return "Search";
+    })();
```

Comment:
Why did you use a self invoking function here? I think you can just inline the content, like so:

```js
const pageTitle = searchQuery && searchQuery.trim()
    ? `${searchQuery} - Search`
    : "Search";
```

I saw you [wrote it](https://github.com/AnimeThemes/animethemes-web/pull/23/files#diff-e0e9dce3eccadfefa994aac0b89f02da9f6e33af82b4207327772857c95799c2R40 ) similar later.

Reply:
Maybe this is simple enough to inline?